modules/configuring-egress-proxy-edns-operator.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-configuring-cluster-wide-egress-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-cluster-wide-proxy_{context}"]
+= Configuring the cluster-wide proxy on the External DNS Operator
+
+You must trust a custom certificate authority (CA) and create the OLM subscription of the External DNS Operator.
```

Comment:
```suggestion
Configuring the External DNS Operator to trust the certificate authority of the cluster-wide egress proxy.
```
`Must trust` is not accurate, the egress proxy may not have the untrusted certificate. Also, to match what we said in ALBO doc.

Reply:
Addressed - maintained the same format as ALBO. Removed the sentence and updated the heading